File: java/matrizes/OrdemDecrescente.java
package matrizes;

import java.util.Scanner;
import java.util.Arrays;

import libraries.TerminalFormating;

public class OrdemDecrescente {
    
    public static void main(String[] args) {
        
        final int REGISTER_NUMBER = 20;

        int[] value = new int[REGISTER_NUMBER];
        int [] value_crescent = new int[REGISTER_NUMBER];

        Scanner input = new Scanner(System.in);
        TerminalFormating terminal = new TerminalFormating();

        //inicio do programa
        terminal.writeProgramTitle("ORDEM DECRESCENTE");

        for(int i = 1; i <= REGISTER_NUMBER; i++) {
            System.out.print("\nDigite o " + i + "º valor: ");
            value[i-1] = input.nextInt();
        }
        
        Arrays.sort(value);

        System.out.println("-----------------------------------");
        for (int i = REGISTER_NUMBER - 1 ; i >= 0; i--) {
            System.out.print(value[i] + " ");
        }
    }
}

File: python/matrizes/ordem_decrescente.py
print("---------------------ORDEM DESCRECENTE----------------------\n")

list_numbers = []

for i in range(20):
    value = int(input(f"Digite o {i+1}º valor: "))
    list_numbers.append(value)

list_numbers.sort(reverse=True)

print("----------------------------------------------")
print("OS NÚMEROS DIGITADOS FORAM")

for element in list_numbers:
    print(element, end=" ")

